lms/templates/dashboard/index.html.jinja2

```diff
@@ -2,7 +2,7 @@
 
 
 {% block title %}
-  {{ assignment.title }} - Hypothesis
+  {{ title }} - Hypothesis
```

Comment:
No need for one template per dashboard, just get the right title on each one.

Reply:
To be honest I reckon this would be much easier to understand if we got the config as parameter here

Now this works because js_config is the context of these views.